contracts/staking/GoodFundManager.sol

```diff
@@ -110,10 +110,10 @@ contract GoodFundManager is DAOContract {
 	 */
 	constructor(NameService _ns) {
 		setDAO(_ns);
-		gdMintGasCost = 140000; // While testing highest amount was 130k so put 140k to be safe
+		gdMintGasCost = 250000; // While testing highest amount was 240k so put 250k to be safe
 		collectInterestTimeThreshold = 5184000; // 5184000 is 2 months in seconds
 		interestMultiplier = 4;
-		gasCostExceptInterestCollect = 650000;
+		gasCostExceptInterestCollect = 850000; //while testing highest amount was 800k so put 850k to be safe
```

Comment:
this is astronomical, why does it cost so much gas?

Reply:
Assume related to too much calculations plus transfers

Reply:
please investigate this.